test/test_helper.bash

```diff
@@ -81,6 +81,25 @@ create_branch_commit() {
   git checkout master
 }
 
+setup_root_git_author() {
+  unset GIT_DUET_AUTHORS_FILE
+  cat > ".git-authors" <<EOF
+---
+authors:
+  dj: Dane Joe
+  fc: Frances Car
+
+email:
+  domain: banana.info.local
+
+email_addresses:
+  dj: dane@bananas.biz.local
+  fc: f.car@banana.info.local
+EOF
+  mkdir new_dir
+  cd new_dir
```

Comment:
Testing that it uses a file in the root directory of the `git` repository instead of a `.git-authors` file in the directory you are currently running `git` commands from inside a repository. I imagined that someone might want multiple nested `.git-authors` files (in the case of a mono repo perhaps) so wanted to explicitly call out and test that it currently does not work like that. Tests work . the same without this so we can remove it if you don't find it helpful.